extensions/amp-list/amp-list.md

```diff
@@ -296,18 +296,19 @@ This attribute specifies an attribute in the returned data that will give the ur
 ```
 
 ### Additional children of `<amp-list>`
```

Comment:
It's intended that we provide UX defaults for these, so when that happens these elements will no longer be mandatory. However, they are mandatory for now.